featuretools/primitives/standard/transform/time_series/box_cox.py

```diff
@@ -0,0 +1,51 @@
+from scipy import stats
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Double
+
+from featuretools.primitives.base import TransformPrimitive
+
+
+class BoxCox(TransformPrimitive):
+    """Applies a transformation to help your data resemble a normal distribution,
+        commonly known as the Box Cox transformation. The input data must be positive.
+        It cannot contain zeroes or negative numbers.
```

Comment:
Can't we offset the data if it has zero or negative values, prior to transforming? Although doing so, would present some of the same challenges that we have with the lambda values.